foundations/html_css/css-foundations.md

```diff
@@ -157,7 +157,9 @@ We have two elements with the `subsection` class that have some sort of unique s
 }
 ~~~
 
-Notice how there isn't any space between the two selectors here. What `.subsection.header` does is it selects any element that has both the `subsection` *and* `header` classes. This basically works with any combination of selectors except multiple type selectors (since an element can't be two different types at once, and `divp` would try to select a literal `<divp>` element which doesn't exist).
+What `.subsection.header` does is it selects any element that has both the `subsection` *and* `header` classes. Notice how there isn't any space between the `.subsection` and `.header` selectors. This syntax basically works for chaining any combination of selectors, with the exception of type selectors.
+
+In general, type selectors can't be chained since an element can’t be two different types at once. Also, chaining two type selectors like `div` and `p`, would give us the selector `divp`, which wouldn't work since the selector would try to find a literal `<divp>` element, which doesn’t exist. 
```

Comment:
I agree the verbiage could be made a little clearer, however I think this could be tweaked a little further. Do you feel something like the below suggestion would ease some of that confusion, as it does seem like the confusion may stem from the "type" word being misconstrued as "you can't use more than one **type of** selector".

```suggestion
What `.subsection.header` does is it selects any element that has both the `subsection` *and* `header` classes. Notice how there isn't any space between the `.subsection` and `.header` class selectors. This syntax basically works for chaining any combination of selectors, with the exception of chaining more than one type (or element) selector.

In general, you can't chain more than one type/element selector since an element can’t be two different types at once. For example, chaining two type selectors like `div` and `p`, would give us the selector `divp`, which wouldn't work since the selector would try to find a literal `<divp>` element, which doesn’t exist. 
```

Reply:
I committed your suggestion, then made another commit where I just made a link to the "Type Selectors" section. Let me know what you think :)